BasicVideoChat/package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "eslint ."
   },
   "dependencies": {
-    "opentok-react-native": "2.28.0",
+    "opentok-react-native": "^2.28.0",
```

Comment:
@jeffswartz should we indicate a specific version, in this case, 2.28.3 since we will update it for each new version?

Reply:
@marinaserranomontes  I think it is best to leave this as "^2.28.0". This is what we already do for each of the sample apps in the repo. This just fixes it for the BasicVideoChat sample. The nice thing about using "^2.28.0" is that we do not have to update this when the SDK version updates (npm will just pull the latest).